front/tests/004-scenario-management.spec.ts

```diff
@@ -17,7 +17,7 @@ test.beforeEach(async () => {
   ({ project, study, scenario } = await setupScenario());
 });
 
-test.describe('Test if operationnal study : scenario creation workflow is working properly', () => {
+test.describe('Test if operational study : scenario creation workflow is working properly', () => {
```

Comment:
I guess ?

```suggestion
test.describe('Test in operational study : scenario creation workflow is working properly', () => {
```

Reply:
Removed the "oprational study"

Reply:
Ty. There is the same sentence in 002 and 003 tests.

Reply:
Done